### DISCUSSION TITLE
Fastapimysql

### DISCUSSION THREAD

**Discussion by velmurugan2020** [fastapi/fastapi#10657] [Questions]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

Fastapi creating no of connection In mysql I can't find the reason,some of the time only it is happening we feel when one query response is delay that time it happening also Some one big query running totally fastapi freezed that I am restarting the service

**Comment by velmurugan2020** [comment]
Hai experts Can I get some support or I need to provide more content?

**Reply by iiotsrc** [reply]
Issues not closed

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean and organized.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
